SquareLine/gen.py

```diff
@@ -0,0 +1,231 @@
+from genericpath import isdir
+import sys
+import os
+import json
+import shutil
+from colorama import Fore
```

Comment:
I also had to install the `colorama` package for this script to work. I think this can be mentioned in the readme file.